nupic/research/frameworks/dynamic_sparse/models/main.py

```diff
@@ -80,7 +80,7 @@ def setup(self):
             )
 
         # add a learning rate scheduler
-        if self.lr_scheduler or self.lr_scheduler == "MultiStepLR":
+        if self.lr_scheduler == "MultiStepLR":
```

Comment:
This change is gonna impact some run files - I've seen it is called StepLR on some files. Are you sure on this?

Reply:
1) The following `elif` statement checks for `self.lr_scheduler == "StepLR"`.
2) Currently, there's a bug: Without removing `or self.lr_scheduler` you'd evaluate `if self.lr_scheduler` to `true` for any non-empty input, and so the scheduler would be invariably `"MultiStepLR"` no matter what you ask for.